public/js/controllers/game.js

```diff
@@ -160,27 +161,27 @@ angular.module('mean.system')
         return false;
       };
       // send invite to users//  
-      $scope.sendInvite = () =>{
-         const gameLink = document.URL; 
-         const usersEmail = $scope.inviteUsers.map((user) => user.email);
-         const message = `${$scope.inviteEmailBody} ${gameLink} `;
-         
-         //backend http request to send emails to invited users
-         $http.post('/api/invite/send', { emails:usersEmail,message:message })
-         .then(() => {
+      $scope.sendInvite = () => {
+        const gameLink = document.URL;
+        const usersEmail = $scope.inviteUsers.map((user) => user.email);
+        const message = `${$scope.inviteEmailBody} ${gameLink} `;
+
+        //backend http request to send emails to invited users
+        $http.post('/api/invite/send', { emails: usersEmail, message: message })
```

Comment:
Consider using object property shorthand for message

Reply:
I didn't work on this file

Reply:
okay